docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -269,6 +272,8 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 180000 | Cannot update a finished event                                                                                                |
 | 180002 | Failed to create stage needed for stage event                                                                                 |
 | 220003 | Webhooks can only create threads in forum channels                                                                            |
+| 200000 | Message was blocked by automatic moderation                                                                                   |
+| 220001 | Title was blocked by automatic moderation                                                                                     |
```

Comment:
```suggestion
| 200000 | Message was blocked by automatic moderation                                                                                   |
| 220001 | Title was blocked by automatic moderation                                                                                     |
| 220003 | Webhooks can only create threads in forum channels                                                                            |
```
numeric order

Reply:
Oh right didn't notice that,
Sorry can't do it rn, just got off of my laptop :(